### PR TITLE
Clicking vis image links to that vis

### PR DESCRIPTION
**Pull request by stowellm**
Addresses #977

### CODE REVIEW

File: app/views/visualizations/index.html.erb [hunk 1]
```diff
@@ -67,9 +67,12 @@
               <% thumb = image_path("vis_#{type.downcase}_light.png")%>
             <% end %>
           <% end %>
-          <div class='item-image'
-               style='background-image:url(<%= thumb %>);background-size: 100% 100%'>
-          </div>
+          <a href='<%= visualization_path vi %>'>
```
Comment (jaypoulz) [inline app/views/visualizations/index.html.erb]: I know I asked for this, but could you instead try this:
```
<a href='<%=visualization_path vi %>'
     class='item-image'
     style='background-image:url(<%= thumb %>);
     background-size: 100% 100%'>
</a>
```

It looks to me as though the div is useless at the moment, unless it's being used for style purposes.

Comment (stowellm) [inline app/views/visualizations/index.html.erb]: Doesn't work.
nope

**Comment by jaypoulz** [comment]
Lol change the ``'`` surround the ``<%`` and it should

**Comment by stowellm** [comment]
Tried multiple variations of it.  Still nothing - no image shows.

**Comment by jaypoulz** [comment]
I done been proven wrong. :smile_cat:

**Pull request merged**